main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -43,14 +43,19 @@ object Kind {
   case object Star extends Kind
 
   /**
-    * The kind of records.
+    * The kind of records row sets.
     */
-  case object Record extends Kind
+  case object RecordRowSet extends Kind
 
   /**
-    * The kind of schemas.
+    * The kind of predicate sets.
     */
-  case object Schema extends Kind
+  case object PredicateSet extends Kind
+
+  /**
+    * The kind of predicates.
+    */
+  case object Predicate extends Kind
```

Comment:
Not needed?

Reply:
My thought was that since the predicate symbols used in schemas are not first-class, they need their own kind. Is that not necessary?

Reply:
I am not sure I understand. Can you expand? What does Predicate correspond to?